### PR TITLE
[SMALLFIX] Minor cleanups

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

File: core/server/src/main/webapp/WEB-INF/master.xml [hunk 1]
```diff
@@ -10,7 +10,12 @@
         <servlet-class>org.glassfish.jersey.servlet.ServletContainer</servlet-class>
         <init-param>
             <param-name>jersey.config.server.provider.packages</param-name>
-            <param-value>alluxio.master;alluxio.master.block;alluxio.master.file;alluxio.master.lineage</param-value>
+            <param-value>
+                alluxio.master
```
Comment (apc999) [inline core/server/src/main/webapp/WEB-INF/master.xml]: just to doublecheck, is this format correct?

**Comment by apc999** [comment]
one question left.

File: core/server/src/main/webapp/WEB-INF/master.xml (hunk 1 above)
Comment (aaudiber) [inline core/server/src/main/webapp/WEB-INF/master.xml]: yes, it makes it easier to scan visually

Comment (apc999) [inline core/server/src/main/webapp/WEB-INF/master.xml]: ack

**Comment by apc999** [comment]
LGTM

**Pull request merged**